CHANGELOG.rst

```diff
@@ -1,9 +1,24 @@
 .. _changelog:
 
+0.33.0
+------
+
+Renku ``0.33.0`` upgrades the PostgreSQL version that is shipped with the project.
+
+NOTE: make sure to check `helm-chart/values.yaml.changelog.md` for instructions on how to
+upgrade to this version of Renku.
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Improvements**
+
+- **Infrastructure**: (BREAKING CHANGE) Upgrade the version of PostgreSQL to 15.3.0. This requires modifying your current values file to work with the new PostgreSQL Helm chart, see (`the helm chart values changelog <https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_) for instructions.
```

Comment:
The wording (BREAKING CHANGE) is confusing to me. What does it refer to? Does it mean that if you do not upgrade then your Renku version will fail?